DEVELOPER_GUIDE.md

```diff
@@ -140,9 +148,14 @@ Run from `bundle-workflow` before making pull requests.
 ```
 cd bundle-workflow
 
+The pre-commit hook checks for imports, type, style and test
+pre-commit run --all-files
+
+Auto-fix format and sort imports by running
```

Comment:
flake8, mypy and pytest don't autofix anything and should be removed now that the precommit hook does the checks